users/users/login.php

```diff
@@ -53,6 +53,12 @@ public function keygen()
 			$id = $model->getUserId('email', $username);            
 		}
 
+		$res					=	array();
```

Comment:
@UmeshBhosale Please go through following document. The response should be same as mentioned here.
http://git.tekdi.net/techjoomla/shika/wikis/rest-api-users

Also, we will also need to do changes in com_api to handle validation errors.